stack/base/before-notebook.d/10_prepare-home-config.sh

```diff
@@ -40,12 +26,23 @@ fi
 mkdir -p --mode=0700 /home/${NB_USER}/.ssh && \
     touch /home/${NB_USER}/.ssh/known_hosts
 
-
 if [[ ! -f  /home/${NB_USER}/.ssh/id_rsa ]]; then
   # Generate ssh key that works with `paramiko`
   # See: https://aiida.readthedocs.io/projects/aiida-core/en/latest/get_started/computers.html#remote-computer-requirements
   ssh-keygen -f /home/${NB_USER}/.ssh/id_rsa -t rsa -b 4096 -m PEM -N ''
 fi
 
-# Start the ssh-agent.
-eval `ssh-agent`
```

Comment:
Based on the discussion, I think we should source the `load-singlesshagent.sh` script here. Otherwise I don't think the ssh-agent will be running when the container starts, since .bashrc is not evaluated AFAIK.

Reply:
Ha! I think the one of the ssh-agent mentioned in https://github.com/aiidalab/aiidalab-mfa-cscs/pull/6 is from here! 
I agree to source the `load-singlesshagent.sh` here. It avoids the duplicate `ssh-agent` I think.

Reply:
Yeah, interesting. That would mean that it is started somewhere else before we get to this script. Maybe the Jupyter image that we use already starts it somewhere. In any case, source the script here is the right thing to do I think.